graphql_compiler/compiler/ir_lowering_match.py

```diff
@@ -726,6 +899,8 @@ def lower_ir(ir_blocks, location_types, type_equivalence_hints=None):
     sanity_check_ir_blocks_from_frontend(ir_blocks)
 
     # These lowering / optimization passes work on IR blocks.
+    optional_locations, location_to_optional, ir_blocks \
```

Comment:
`\` should be avoided pretty much always. Instead, assign the result value to an intermediate variable, then destructure it on the next line.